server/controllers/importers/sif.js

```diff
@@ -11,26 +11,21 @@ module.exports = function (sif) {
     var warnings = [];
     var errors = [];
 
-    // Empty SIF files must return a network, thus must contain some data.
-    if (!sif) {
-        sif = " ";
-    }
+    // Workaround for empty SIF file handling
+    var emptySifFile = sif === "";
+    sif =  sif ? sif : " ";
```

Comment:
This type of conditional expression is the same as the shorter:

    sif = sif || " ";